debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -431,18 +431,36 @@ private String asHstoreString(byte[] data) {
 
     private Object convertHstoreToJsonString(Column column, Field fieldDefn, Object data){
         return convertValue(column, fieldDefn, data, "{}", (r) -> {
+          logger.debug("in ANON: value from data object: ***" + data.toString() + "***");
```

Comment:
Please use `logger.debug("Text {} from log", variant);` variant of logging methods. Also the `trace` level would be better for most og the messages

Reply:
I believe all the debug statements have been moved to trace, and I have updated to use the `logger.trace("{}", variable)` method